guides/source/5_2_release_notes.md

```diff
@@ -147,35 +456,399 @@ Please refer to the [Changelog][action-mailer] for detailed changes.
+
+*   Fix to invoke callbacks when using `update_attribute`.
+    ([Commit](https://github.com/rails/rails/commit/732aa34b6e6459ad66a3d3ad107cfff75cc45160))
+
+*   Use `count(:all)` in `HasManyAssociation#count_records` to prevent invalid
+    SQL queries for association counting.
+    ([Pull Request](https://github.com/rails/rails/pull/27561))
+
+*   Fixed inconsistency with `first(n)` when used with `limit()`.
+    The `first(n)` finder now respects the `limit()`, making it consistent
+    with `relation.to_a.first(n)`, and also with the behavior of `last(n)`.
+    ([Pull Request](https://github.com/rails/rails/pull/27597))
+
+*   Fix nested `has_many :through` associations on unpersisted parent instances.
+    ([Commit](https://github.com/rails/rails/commit/027f865fc8b262d9ba3ee51da3483e94a5489b66))
+
+*   Take into account association conditions when deleting through records.
+    ([Commit](https://github.com/rails/rails/commit/ae48c65e411e01c1045056562319666384bb1b63))
+
+*   Don't allow destroyed object mutation after `save` or `save!` is called.
+    ([Commit](https://github.com/rails/rails/commit/562dd0494a90d9d47849f052e8913f0050f3e494))
+
+*   Fix relation merger issue with `left_outer_joins`.
+    ([Pull Request](https://github.com/rails/rails/pull/27860))
+
+*   Support for PostgreSQL foreign tables.
+    ([Pull Request](https://github.com/rails/rails/pull/31549))
+
+*   Clear the transaction state when an Active Record object is duped.
+    ([Pull Request](https://github.com/rails/rails/pull/31751))
+
+*   Fix `count(:all)` with eager loading and having an order other than
+    the driving table.
+    ([Commit](https://github.com/rails/rails/commit/ebc09ed9ad9a04338138739226a1a92c7a2707ee))
+
+*   Fix not expanded problem when passing an Array object as argument
+    to the where method using `composed_of` column.
+    ([Pull Request](https://github.com/rails/rails/pull/31724))
+
+*   PostgreSQL: Allow pg-1.0 gem to be used with Active Record.
+    ([Pull Request](https://github.com/rails/rails/pull/31671))
+
+*   Make `reflection.klass` raise if `polymorphic?` not to be misused.
+    ([Commit](https://github.com/rails/rails/commit/63fc1100ce054e3e11c04a547cdb9387cd79571a))
+
+*   Fix `#columns_for_distinct` of MySQL and PostgreSQL to make
+    `ActiveRecord::FinderMethods#limited_ids_for` use correct primary key values
+    even if `ORDER BY` columns include other table's primary key.
+    ([Commit](https://github.com/rails/rails/commit/851618c15750979a75635530200665b543561a44))
+
+*   Fix that after commit callbacks on update does not triggered
+    when optimistic locking is enabled.
+    ([Commit](https://github.com/rails/rails/commit/7f9bd034c485c2425ae0164ff5d6374834e3aa1d))
+
+*   Fix `dependent: :destroy` issue for has_one/belongs_to relationship where
+    the parent class was getting deleted when the child was not.
+    ([Commit](https://github.com/rails/rails/commit/b0fc04aa3af338d5a90608bf37248668d59fc881))
```

Comment:
Is it intentional to refer to backported commits rather than PRs?

Reply:
Yes, since it is related to 5.2.